### DISCUSSION TITLE
How do I share security vulnerabilities found in the longhorn container images?

### DISCUSSION THREAD

**Discussion by eldada** [longhorn/longhorn#3415] [Q&A] [answered]
Hi team. I used JFrog Artifactory to upload a private copy of the longhorn images for a private testing environment I'm working on.
While doing so, the images were scanned by JFrog Xray and a long list of security vulnerabilities was reported.

How and who can I share the report with?

**Comment by jenting** [comment]
cc @innobead

**Comment by innobead** [comment, marked as answer]
@eldada for any vulnerabilities, please send to longhorn-security@suse.com. Thank you.

**Reply by eldada** [reply]
Sent

**Reply by eldada** [reply]
I got the following:
```
Your message wasn't delivered to Longhorn-Security@suse.com because the address couldn't be found, or is unable to receive mail.
```

**Reply by innobead** [reply]
Try again, it should work.

**Reply by eldada** [reply]
Sent
